File: ProAgil.Domain/Palestrante.cs
using System;
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Palestrante
    {
        public int ID { get; set; }
        public string Nome { get; set; }
        public string Miniurriculo { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public List<RedeSocial> RedeSociais { get; set; }
        public List<PalestranteEvento> PalestrantesEventos { get; set; }       

    }
}

File: ProAgil-App/src/app/models/Palestrante.ts
import { RedeSocial } from "./RedeSocial";
import { Evento } from "./Evento";

export class Palestrante {

    constructor(){}

    ID: number; 
    Nome: string;
    Miniurriculo: string;
    ImagemURL: string;
    Telefone: string;
    Email: string;
    RedeSociais: RedeSocial[];
    PalestrantesEventos: Evento[];
}
